src/main/java/hudson/plugins/git/GitSCM.java

```diff
@@ -1068,16 +1068,20 @@ public void checkout(Run<?, ?> build, Launcher launcher, FilePath workspace, Tas
 
         try {
           checkoutCommand.execute();
-        } catch(GitLockFailedException e) {
+        } catch (GitLockFailedException e) {
```

Comment:
I.e. I just spent twenty minutes fighting intellij to get it to stop trying
to fix all that in this commit and spew whitespace changes all over the
place. Oy, the pain!